docs/index.md

```diff
@@ -367,6 +367,27 @@ Adding errors to action object is important, because each time a handled excepti
 Validation exception will have the same backtrace as original error.
 Prefer this way over custom exception handling in private methods.
 
+### Dependency Injection
+
+Granite uses action initializer for attributes.
+However, we can still manually inject dependencies.
+
+```ruby
+class Action < Granite::Action
+  attribute :name, String
+
+  private attr_reader :my_dep
```

Comment:
Optional, but I think the example could be minified by removing this line since it's just basic ruby and not needed for the purpose of this example.